.github/workflows/ci.yml

```diff
@@ -6,6 +6,23 @@ on:
   pull_request:
 
 jobs:
+  type_check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Install Mypy Dependencies
+        run: pip install mypy
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously. -->
**🎯 suggestion:** let's add this to the `test` dependencies if anyone wants to run this themselves, and then we can install similar to the `tests` job below 

https://github.com/Learnosity/learnosity-sdk-python/blob/3784db73543054770e8c6dec891364aa2cf42e85/setup.py#L21-L26

https://github.com/Learnosity/learnosity-sdk-python/blob/c809c6630df11f64cdada13bb31342a5fa75f21b/.github/workflows/ci.yml#L23-L24

Reply:
Thanks for addressing 👍